.github/workflows/ci-Exporter.OneCollector-Integration.yml

```diff
@@ -46,13 +46,13 @@ jobs:
     - name: Setup dotnet
       uses: actions/setup-dotnet@v4
 
-    - name: dotnet restore build/Projects/${{env.PROJECT}}.proj
-      run: dotnet restore build/Projects/${{env.PROJECT}}.proj
+    - name: dotnet restore Component.proj[${{ env.BUILD_COMPONENT }}]
+      run: dotnet restore build/Projects/Component.proj
 
-    - name: dotnet build build/Projects/${{env.PROJECT}}.proj
-      run: dotnet build build/Projects/${{env.PROJECT}}.proj --configuration Release --no-restore
+    - name: dotnet build Component.proj[${{ env.BUILD_COMPONENT }}]
+      run: dotnet build build/Projects/Component.proj --configuration Release --no-restore
 
-    - name: dotnet test test/${{env.PROJECT}}.Tests
-      run: dotnet test test/${{env.PROJECT}}.Tests --filter CategoryName=OneCollectorIntegrationTests --framework ${{ matrix.version }} --configuration Release --no-restore --no-build --logger:"console;verbosity=detailed"
+    - name: dotnet test Component.proj[${{ env.BUILD_COMPONENT }}]
```

Comment:
I do not fully understand the syntax here, first time I see `[` `]` in this context. Do you have any documentation with explanation what is the result of call
`Component.proj[${{ env.BUILD_COMPONENT }}]`?

Reply:
@Kielek 

I refactored things to try and make it more clear what is going on. Check out this commit: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1811/commits/77191cab2bd582d63b384029508fe7bf1c19a263

LMK if you feel it needs more work.

I'm happy to explain here but I figured making the code more self-explanatory would be better in the long run.

Reply:
Nice! Now it is clear to me.